DPG/Tasks/AOTTrack/qaMatchEff.cxx

```diff
@@ -404,6 +407,20 @@ struct qaMatchEff {
     //
     for (auto& jT : jTracks) {
 
+      // choose if we keep the track according to the TRD presence requirement
+      if (b_TRDisThere && !jT.hasTRD())
+        continue;
+      if (!b_TRDisThere && jT.hasTRD())
```

Comment:
In my opinion we do need to add a case in which we do require the TRD, but not a case in which we explicitly remove it. In other words, in my opinion we should not ask explicitly to remove the TRD, but just to see what changes if we do explicitly require it or not. Then, in my opinion this second `if` here shall be removed (here and below as well)